feature_engine/feature_selection.py

```diff
@@ -0,0 +1,69 @@
+# Authors: Soledad Galli <solegalli@protonmail.com>
+# License: BSD 3 clause
+
+import numpy as np
+import pandas as pd
+import warnings
+
+from sklearn.base import BaseEstimator, TransformerMixin
+from feature_engine.dataframe_checks import _is_dataframe
+from feature_engine.variable_manipulation import _define_variables
+
+
+class FeatureEliminator(BaseEstimator, TransformerMixin):
```

Comment:
is the name FeatureEliminator clear enough? 
I think I would prefer to call the transformer **DropFeatures**? 

Eliminator gives me the sensation that there is some sort of fancy algorithm running, and we are only removing the features that the user inputs. What do you think?

Reply:
I am very bad in naming things so, I think DropFeatures makes sense. If you're okay with DropFeatures I will go ahead and rename it.